docs/storage/petalibrary/rclone.md

```diff
@@ -2,12 +2,9 @@
 
 This guide details the process of making a backup for local data on your laptop or lab server to a PetaLibrary allocation. The method employs the software _rclone_, which is a command line application that is available for many architectures. While the following tutorial is tailored for a MacOS user, the general steps to follow are the same and are relevant for Windows and Linux users too.
 
-## Generate public/private keypair for CURC resources
+## Generate public/private keypair for CURC resources (if you wish to not enter a password each time you transfer files)
```

Comment:
This title is way too long. Please replace it with something shorter.

Reply:
Done